### ISSUE TITLE
Appimage no longer auto updates

### ISSUE THREAD

**Issue by MNarath1** [bitwarden/clients#7894]
### Steps To Reproduce

1. Open Bitwarden Version 2024.1
2. Click on Search for update
3. Response you are on the newest Version

### Expected Result

Getting a Prompt to Update to the newest Version.

### Actual Result

No update Response at all it just says newest Version

### Operating System

Linux

### Operating System Version

Linux MX 23 KDE

### Installation method

Direct Download (from bitwarden.com)

### Build Version

Bitwarden-2024.1.0

### Issue Tracking Info

- [X] I understand that work is tracked outside of GitHub. A PR will be linked to this issue should one be opened to address it, but Bitwarden doesn't use fields like "assigned", "milestone", or "project" to track progress.

**Comment by MNarath1** [comment]
After checking the logs on a device with a new install i found this in the logs:
`[2024-02-09 19:43:00.153] [info]  Update for version 2024.1.0 is not available (latest version: 2024.2.0, downgrade is disallowed).`

**Comment by MNarath1** [comment]
The issues has occured on a Steamdeck, MX Linux OS but it seems to work fine Ubuntu for some reason.

**Comment by MNarath1** [comment]
just to be safe i confirmed that this issue happens even on the live usb i used to install linux mx (not touched in months ) and yes but i have no idea how.

**Comment by MNarath1** [comment]
Possible another maybe important detail might be that i am using KDE Plasma as my Desktop enviroment (Basically the only thing that i could think of that my MXlinux install and steamdeck have in common)

**Comment by sammbw** [comment]
Hi there,

Thank you for your report! 

I was able to reproduce this issue, and I have flagged this to our engineering team.

If you wish to add any further information/screenshots/recordings etc., please feel free to do so at any time - our engineering team will be happy to review these.

Thanks once again!

**Comment by valtoree** [comment]
Yes me too. On linux mint xfce, on bitwarden version 2024.2.0 and when i hit update, tells me no updates currently available. Even though I can see on github 2.1 has being released 3 days ago.

**Comment by wodin** [comment]
I'm getting the same issue:

```
Version 2024.10.2
Shell 32.1.1
Renderer 128.0.6613.137
Node 20.17.0
Architecture x64
```

If I run it from the command line, I get the following output:
```
...
13:39:48.461 › Update for version 2024.10.2 is not available (latest version: 2024.11.0, downgrade is disallowed).
```

**Comment by wodin** [comment]
I see there's an update available today, but the app still thinks it would be a downgrade.
```
13:28:40.244 › Update for version 2024.11.0 is not available (latest version: 2024.11.1, downgrade is disallowed).
```

**Comment by trmartin4** [comment]
@MNarath1 This is a known issue with the AppImage package format.  We have recently updated our desktop [download page](https://bitwarden.com/download/#downloads-desktop) to more accurately reflect the capabilities of each of the builds that we offer, including a specific callout for AppImage.

As this is an upstream issue inherent to the format itself, we will be closing this ticket.  The download page offers a few different recommendations for third-party tools that can be used to bridge this gap, but none of them are officially supported or audited by Bitwarden.

![image](https://github.com/user-attachments/assets/a043ec00-151f-45d3-a46e-671da49f63f0)
